### PR TITLE
Add dynamodb client and refactor usages of json file from scripts

### PR DESCRIPTION
**Pull request by VKong6019**
Issue: https://github.com/sandboxnu/boba-buddies/issues/29

HI i took the liberty in adding AWS sdk to include the DynamoDB client and refactor some usages of the `pairings.json` file from the manual scripts. Most of the logic for interacting with the DynamoDB tables we have are in the `usersDataManager.ts` and `pairsDataManager.ts` that pretty much includes some basic CRUD methods.

- the user manager will interact with a DynamoDB table - `boba-buddies-users` - that tracks the current list of members in the boba-buddies channel.
- the pairs manager will interact with a DynamoDB table - `boba-buddies-pairs` - that tracks the current list of channel ids and pairings from the boba-buddies channel.

~**IMPORTANT**: the refactoring part hasn't been tested yet so I will need to do that tomorrow and this will be fully ready for review~

### CODE REVIEW

File: database/dbConfig.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+require('dotenv').config();
+
+AWS.config.update({
+	region: process.env.DB_DEFAULT_REGION,
+	accessKeyId: process.env.DB_ACCESS_KEY_ID,
+	secretAccessKey: process.env.DB_SECRET_ACCESS_KEY
+});
+
+export const DB_CLIENT = new AWS.DynamoDB.DocumentClient();
+const POOL_TABLE: string = "boba-buddies-pool";
```
Comment (VKong6019) [inline database/dbConfig.ts:11]: this line can be ignored. im going to move this into a new data manager in a different pr

File: database/usersDataManager.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import { DB_CLIENT } from "./dbConfig";
+
+const USERS_TABLE: string = "boba-buddies-users";
```
Comment (VKong6019) [inline database/usersDataManager.ts:3]: im keeping track of the slack users in the boba-buddies channel for the pr i will make on generating pairs to account for people who join halfway thru the semester

**Comment by VKong6019** [comment]
cool beans, i just tested it with the scripts and fix some bugs. i think it should be good to go after review!

**Review by emaela — APPROVED** [review]
It looks good to me!

**Pull request merged**